sbnana/SBNAna/Cuts/NumuCutsIcarus202208.cxx

```diff
@@ -0,0 +1,51 @@
+#include "sbnana/CAFAna/Core/Cut.h"
+#include "sbnana/SBNAna/Cuts/NumuCutsIcarus202208.h"
+#include "sbnana/SBNAna/Vars/NumuVarsIcarus202208.h"
+
+#include "sbnanaobj/StandardRecord/Proxy/FwdDeclare.h"
```

Comment:
No need for the fwd declare when you're actually including the whole thing on the next line anyway.